### ISSUE TITLE
Delete single item from quickfix list

### ISSUE THREAD

**Issue by timsofteng** [romainl/vim-qf#117]
Hello. Can this plugin help me to delete single item from quickfix list?

**Comment by romainl** [comment]
Not at the moment. I have had some discussion about this with other users in the past but it sadly went nowhere.

The ability to do something like `:.Reject` to reject the current line or `:11,15Reject` to reject lines 10 to 15 (actually entries 9 to 14) would probably be useful.

**Comment by timsofteng** [comment]
@romainl Yep. It will be pretty nice if user can delete current item from itself or any other item from quickfix list buffer.
I don't understand why it's not built-in functionality.

**Comment by romainl** [comment]
Vim now has `:help :Cfilter`, which is roughly equivalent to vim-qf's now deprecated `:Filter` and works pretty much the same way. Maybe it actually is worth it to open a feature request for range support over at vim/vim?

**Comment by timsofteng** [comment]
> Vim now has `:help :Cfilter`, which is roughly equivalent to vim-qf's historic `:Filter` and works pretty much the same way. Maybe it actually is worth it to open a feature request for range support over at vim/vim?

Maybe it makes sense.
Could you open it in vim repo? I'm sure you could describe problem better than me as a developer.

**Comment by romainl** [comment]
I just pushed range support for `:Keep` and `:Reject`.

You can now do `:.Reject` to remove the currently selected entry or `:10,15Reject` to remove entries 10-15. Same for `:Keep`.

Happy to get any feedback.
